app/src/main/java/com/example/pinterbest/data/repository/Repository.kt

```diff
@@ -45,7 +47,7 @@ class Repository(val apiService: ApiService) {
         }
     }.flowOn(Dispatchers.IO)
 
-    fun getpinFeed() = result {
-        apiService.getPinFeed()
+    fun getpinFeed(context: Context) = result {
```

Comment:
context in repositories is prohibited. Repository must know nothing about Android framework

Reply:
Fixed here https://github.com/enginire4ek/Pinterbest/pull/3/commits/f47063b96f0a24a56ad6def81896bbe4dee63167